### DISCUSSION TITLE
v14 documentation

### DISCUSSION THREAD

**Discussion by wouterds** [invertase/react-native-firebase#6851] [Q&A]
I am like many others stuck on v14 for reasons, I am looking for the v14 documentation though as it's not the same. Can I still find it somewhere in a readable format? Or is it possible to add versions to the docs (e.g. like [`react-navigation`](https://reactnavigation.org/) does)?

**Comment by traxx10** [comment]
I'm also looking for v14 documentation.
Can't upgrade because of flipper

**Comment by chichilatte** [comment]
You have the github repo for v14.11.1 (committed on 17 June 2022)...
https://github.com/invertase/react-native-firebase/tree/b1a430bbf36a41076b33550fec113fa0e44583b4
(the version number is in `lerna.json`)

And you can find v14 docs on the Wayback Machine (originally on rnfirebase.io)...
https://web.archive.org/web/20220530071115/https://rnfirebase.io/
The nearest date they have for v14 is 30 May 2022.
_Note: They have a 15 June 2022 one, but it [seems to be](https://web.archive.org/web/20220615194022/https://rnfirebase.io/#altering-cocoapods-to-use-frameworks) for v15._
